src/controllers/authentication.js

```diff
@@ -16,15 +18,42 @@ const createUser = async (req, res) => {
   };
   try {
     const user = await models.User.create(userData);
-    const {password, ...data} = user.toJSON();
-    const token = tokenGenerator({ userId: user.id });
+    const token = tokenGenerator({ userId: user.id }, { expiresIn: "1d" });
+    const url = `http://localhost:4000/user/verify_email/${token}`;
+    sendMail(
```

Comment:
Please add this link in `.env` since it will change according to the environment we are on
```javascript
 const url = `${process.env.BASE_URL}/user/verify_email/${token}` 
```